requirements_dev.txt

```diff
@@ -1,28 +1,11 @@
-argh==0.26.2
-attrs==18.1.0
-bumpversion==0.5.3
-certifi==2018.4.16
-chardet==3.0.4
-coverage==4.5.1
-flake8==3.5.0
-idna==2.7
-mccabe==0.6.1
-pathtools==0.1.2
-pkginfo==1.4.2
-pluggy==0.6.0
-py==1.5.4
-pycodestyle==2.3.1
-pyflakes==1.6.0
-pytest==3.4.2
-pytest-runner==2.11.1
-PyYAML>=4.2b1
-requests==2.20.1
-requests-toolbelt==0.8.0
-singletonify==0.1.2.0
-six==1.11.0
-tox==2.9.1
-tqdm==4.23.4
-twine==1.11.0
-urllib3==1.23
-virtualenv==16.0.0
-watchdog==0.8.3
+# pip install ocean-brizo
```

Comment:
It's a minor thing, but here and again below, `ocean-brizo` should be replaced with the name of the package in PyPI, i.e. with `osmosis-driver-interface`

The same in `setup.py` (just do a search for `ocean-brizo` across the project.

Reply:
Updated. What an eagle-eyed ;)